_projects/scalems.md

```diff
@@ -4,9 +4,21 @@ collection: projects
 permalink: /projects/scalems
 abstract: "Supporting algorithms that use large collections of molecular simulations to simulate the long timescales needed for scientific and industrial discovery."
 status: active
+logo: mgray_800x600.png
 ---
 
-SCALE-MS will develop computing tools to simplify the process of writing algorithms that use large collections of molecular simulations to simulate the long timescales needed for scientific and industrial understanding. These tools will make it much simpler to have simulations interact adaptively, so simulation results can automatically guide the creation and running of new simulations. By making these complex multi-simulation algorithms easier to create and run, this project will enable users to run existing methods in computational molecular science more easily and make it possible for researchers to create and test new, even more powerful, methods for molecular modeling. This project brings together researchers from biophysics, chemical engineering, materials science, and software engineering.
-
-<a href="https://github.com/radical-collaboration/SCALE-MS"><i class="fa fa-github"></i>GitHub Repositories</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1835449"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/mgray_800x600.png" width="150">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```